app/resources/view.html.template

```diff
@@ -265,7 +264,7 @@
 
             {% if linked_person_info %}
               <div class="other section">
-                <h3>{% trans "Possible duplicates" %}</h3>
+                <h2>{% trans "Possible Duplicates" %}</h2>
```

Comment:
Other <h2> titles are sentence cased. We should keep this consistent, too, unless you feel strongly about changing it to title cased. Changing message will require re-translation using Translation Console, which will take weeks for minor languages. Unless we have a strong reason, we shouldn't change messages.